n3c_owl_ingest/n3c_owl_ingest.py

```diff
@@ -1,4 +1,14 @@
 """N3C OMOP to OWL"""
+import os
+from pathlib import Path
+
+SRC_DIR = Path(os.path.dirname(os.path.abspath(__file__)))
+PROJECT_DIR = SRC_DIR.parent
+TERMHUB_CSETS_DIR = PROJECT_DIR / 'termhub_csets'
+DATASETS_DIR = TERMHUB_CSETS_DIR / 'datasets' / 'prepped_files'
+TARGET_DATASETS = [
+
+]
 
 
 def run_ingest():
```

Comment:
@Sigfried @ShahimEssaid Just tagging you guys here as an initial FYI. Not putting much more time into this this week. I believe most of this work will be billed under TermHub time, not TIMS. Or I could do 50/50, not sure.

@Sigfried I felt like this was the best place to put this ingest; with all the other TIMS ingests in `HOT-Ecosystem`. Right now I'm cloning `termhub-csets` and pulling the datasets directly from that. I have ideas on how to make this better, e.g. (a) `pip install enclave-wrangler` and pulling datasets from that, (b) or automating releases in `termhub-csets` through a GitHub action, but that is not my priority at this stage.